variants/heltec_vision_master_e213/pins_arduino.h

```diff
@@ -3,7 +3,7 @@
 
 #include <stdint.h>
 
-static const uint8_t LED_BUILTIN = -1; // Board has no built-in LED, despite what schematic shows
+static const uint8_t LED_BUILTIN = 45; // Board has no built-in LED, despite what schematic shows
```

Comment:
Is that comment still valid after this change?

Reply:
Good spotting!